lib/src/crop.dart

```diff
@@ -76,6 +75,12 @@ class Crop extends StatelessWidget {
   /// If default dot Widget with different color is needed, [DotControl] is available.
   final CornerDotBuilder? cornerDotBuilder;
 
+  /// [Color] of the image border
+  /// When [borderColor] is set, a border with the given color is added to the
+  /// image exterior, making it better to see the image ending when
+  /// the [baseColor] and the image background are similar
+  final Color? borderColor;
```

Comment:
Why don't we accept `Border`, not only `Color`?
It enables users to decide other configurations, such as width.